tests/ast/test_ast_driver.py

```diff
@@ -222,6 +224,10 @@ def compare_base64_results(
         clear_symbols(actual_message)
         clear_symbols(expected_message)
 
+    if exclude_udfs:
```

Comment:
I think we can unify the `exclude_udfs` and `exclude_symbols_and_src` flags into a `decoder` single decoder flag for the comparator